### ISSUE TITLE
Clarifying definition and purpose of validation set

### ISSUE THREAD

**Issue by priyald17** [390introml/introml-notes-tex#13]
In Chapter 2 (Regression), we give the following mental model:
- Training data: Use it to fit the parameters. If you want to fit hyper-parameters, do cross-validation on the training data
- Validation data: "Fake unseen data" that you can use to see how well your hypothesis generalizes
- Test data: Unseen data

However, a different (common) usage of terms is:
- Training data: Use it to fit the parameters
- Validation data: What you use to fit the hyper-parameters
- Test data: "Fake unseen data" where you can see how well your hypothesis generalizes.

It might be good to disambiguate this. Otherwise, after the class, students may end up in a situation where they conflate the two mental models - e.g. by using the validation set to fit hyperparameters [definition 2] but then also using it to evaluate the generalization of their hypothesis [definition 1]

**Comment by priyald17** [comment]
Related comment from one of the course TAs: They felt that training vs. validation vs. test data was introduced in Recitation 1, but then after that, we only really ever talk about training data vs. test data. As such, the "importance" of validation data did not come through as much as it could have in the course.

**Comment by shensquared** [comment]
Thanks Priya!

Re: alternative definition. Might be good to add as a margin/side note. I actually think the wording in that main text could also be clarified/improved.


Re: also sorta agreed. My gut feeling is probably because other week-specific takeaways took center stage; but I'm sure there're places we could hit this validation note again. I'll add this on to HW feedback.
